jha/src/views/enrolment/FPCareEligibilityPage.vue

```diff
@@ -2,12 +2,57 @@
   <div>
     <PageContent :deltaHeight='pageContentDeltaHeight'>
       <div class="container pt-3 pt-sm-5 mb-3">
-        <h1>Eligibility questionnaire</h1>
+        <h1>Eligibility Questionnaire</h1>
+        <p class="mb-0">You can use the Joint Health Application to apply for Medical Services Plan (MSP) enrolmnent, Fair PharmaCare, and/or MSP Supplementary Benefits.</p>
+        <p class="mt-0">Answering the following questions will help you determine which programs you are eligible for, and ensure you have the required documents and information to complete your application(s).</p>
         <hr class="mt-0"/>
-        <h2>Fair PharmaCare</h2>
+        <h2>Fair PharmaCare Enrolment</h2>
+        <Radio
+          id='apply-fpc'
+          name='apply-fpc'
+          label='1. Will you use the Joint Health Application to register for Fair PharmaCare?'
+          v-model='applyFPC'
+          :items='radioOptionsApplyFPC' />
```

Comment:
The `Radio` usages are missing `blur` handlers. They need to validate the fields upon tabbing out.

Reply:
They don't need to validate at all... the only validation required for the questionnaires is to stop the user being able to continue if they haven't answered the last question asked.  That error should only be presented if they click continue.  Therefore blur handler is not needed.